src/lnp/application/payment/types.rs

```diff
@@ -25,8 +26,12 @@ use bitcoin::OutPoint;
 use crate::bp::chain::AssetId;
 use crate::bp::Slice32;
 use crate::lnp::application::extension;
-use crate::strict_encoding::{self, strict_deserialize, strict_serialize};
-
+use crate::lnp::presentation::encoding::{
+    Error as ln_error, LightningDecode, LightningEncode,
```

Comment:
Lowercase names for types are not desired. You may use `as LightningError`

Reply:
Done.